src/test/functional/tests/addUser.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { expect, test } from '@playwright/test';
+
+import { adminUserTestData } from '../data';
+import { LoginPage } from '../pages/LoginPage';
+import { PimPage } from '../pages/PimPage';
+
+let loginPage: LoginPage, pimPage: PimPage;
+
+test.describe('Adding a new employee', () => {
+  test.beforeEach(async ({ page }) => {
+    loginPage = new LoginPage(page);
+    pimPage = new PimPage(page);
+    await loginPage.initialize();
+    await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password);
+    await loginPage.navigateToSubPage('PIM');
+    await expect(page.getByRole('heading', { name: 'Employee Information' })).toBeVisible();
+  });
+
+  test.afterEach(async () => {
+    await loginPage.close();
+  });
+
+  test('Add new employee user with account  & fill new employee personal details', async () => {
+    await pimPage.addEmployeeWithLoginCredentials();
+    await pimPage.fillNewEmployeePersonalDetails();
```

Comment:
I advise a good approach (when possible) of GWT (given, when, then).
It could go like that:
```
//given
const newUserData = getDefaultNewUserData();

//when
await pimPage.addEmployeeWithLoginCredentials(newUserData);
await pimPage.fillNewEmployeePersonalDetails();

//then
const actualUserData = await somePageOrClass.fetchUserData(newUserData.username);
await expect(actualUserData).toBe(newUserData)
```
or if a password can't be fetched some comparator can be made which ignores password in that case

Reply:
For now I will leave it as it it. Later, maybe it is good idea to do that. Thanks